README.md

```diff
@@ -152,7 +152,7 @@ import { hasZodFastifySchemaValidationErrors } from 'fastify-type-provider-zod'
 fastifyApp.setErrorHandler((err, req, reply) => {
     if (hasZodFastifySchemaValidationErrors(err)) {
         return reply.code(400).send({
-            error: 'Response Validation Error',
+            error: 'Schema Validation Error',
```

Comment:
wouldn't it technically be Response Schema Validation Error?

Reply:
No it validates the request: body, params and querystring, not the response. That is ResponseSerializationError.

Reply:
right. Wouldn't `Request validation error` be more intuitive then?

Reply:
Also possible, but fastify calls it `FastifySchemaValidationError`. Both will work